app/src/main/java/org/simple/clinic/summary/PatientSummaryScreenController.kt

```diff
@@ -129,10 +147,13 @@ class PatientSummaryScreenController @Inject constructor(
         .map { (patient, address, phoneNumber) -> { ui: Ui -> ui.populatePatientProfile(patient, address, phoneNumber) } }
   }
 
-  private fun constructListDataSet(events: Observable<UiEvent>): Observable<UiChange> {
+  private fun constructListDataSet(events: Observable<UiEvent>): Observable<PatientSummaryItemChanged> {
     val patientUuids = events
         .ofType<PatientSummaryScreenCreated>()
         .map { it.patientUuid }
+        .distinctUntilChanged()
```

Comment:
Is this necessary since it
1. Always will emit only once.
2. Will never change at all.

Reply:
Since `PatientSummaryScreenCreated` event gets emitted every time we pass `replayedEvents` to various streams, we need the `distinctUntilChanged()` to avoid the individual item streams from emitting multiple times.